data/mods/blazemod/vehicleparts/blob_vehicleparts.json

```diff
@@ -0,0 +1,725 @@
+[
```

Comment:
Painstakingly separated out, this is the actual part versions of the parts, unchanged. I could use templates to trim these down as well, and some of the fields definitely need modernising, but I'll leave them be for now.